src/components/pages/bookstore/BookstorePage.js

```diff
@@ -0,0 +1,20 @@
+import { useState } from "react"
+import { BooksListContext } from "../../../context/BooksListContext"
+import { SearchMenu } from "../../searchMenu/SearchMenu"
+import { StyledBooksListElement } from "./StyledBooksListElement"
+import { StyledDecorationBar } from "../../decorationBar/StyledDecorationBar"
+
+export const BookstorePage = () => {
+    const [booksList, setBooksList] = useState([])
+    return (
+        <main>
+            <BooksListContext.Provider value={{
```

Comment:
Wydzielić to do globalnych providerów i opakować to widok dla którego chcemy go zastosować

Reply:
@bwolak94 o coś takiego chodzi?
![image](https://user-images.githubusercontent.com/26098242/191343298-d6e5fa8b-95a0-438b-b934-b6dc5134bef2.png)